python/tskit/trees.py

```diff
@@ -10476,6 +10476,30 @@ def ld_matrix(
             mode=mode,
         )
 
+    @property
+    def individual_nodes(self):
+        """
+        Return an array of node IDs for each individual in the tree sequence.
+        Errors if the tree sequence has no individuals.
+
+        :return: Array of shape (num_individuals, max_ploidy) containing node IDs.
+            Values of -1 indicate unused slots for individuals with ploidy
+            less than the maximum.
+        :rtype: numpy.ndarray (dtype=np.int32)
+        """
+        if self.num_individuals == 0:
```

Comment:
I agree, this isn't an error, and should just return an empty array (ideally without a special case for C code)

Reply:
Done in 4a815ba6